src/core/network/qgsfiledownloader.cpp

```diff
@@ -101,9 +101,12 @@ void QgsFileDownloader::onSslErrors( QNetworkReply *reply, const QList<QSslError
     QStringList errorMessages;
     errorMessages.reserve( errors.size() + 1 );
     errorMessages <<  QStringLiteral( "SSL Errors: " );
-    for ( auto end = errors.size(), i = 0; i != end; ++i )
```

Comment:
These are odd loops - they actually aren't looping in reverse like your change does